<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkAddressbookUI_Base</class>
 <widget class="QWidget" name="LinkAddressbookUI_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="3" column="1">
    <spacer name="spacer11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>405</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="addAddresseeButton">
     <property name="toolTip">
      <string>Create a new entry in your address book</string>
     </property>
     <property name="text">
      <string>Create New &amp;Entry...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblHeader" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTreeWidget" name="addresseeListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Select the contact you want to communicate with via Instant Messaging</string>
     </property>
     <property name="iconSize">
      <size>
       <width>1080</width>
       <height>60</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Photo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="lblSearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>S&amp;earch:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>kListViewSearchLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KTreeWidgetSearchLine" name="kListViewSearchLine"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <customwidgets>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kactivelabel.h</include>
  <include location="local">ktreewidgetsearchline.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
